src/main/perl/Path.pm

```diff
@@ -544,6 +565,103 @@ sub directory
 }
 
 
+=item _make_link
+
+This method is mainly a wrapper over C<LC::Check::link>
+returning the standard CAF::Path return values. Every option
+supported by C<LC::Check::link> is supported. C<NoActionFlag>
```

Comment:
`C<NoActionFlag>` -> `C<NoAction>-flag`; also mention that it support the `keeps_state` option